src/zentables/zentables.py

```diff
@@ -840,53 +832,63 @@ def _combine_n_pct(
     return df_n_str.add(df_pct_str).fillna(f"{suppress_symbol}")
 
 
-def _is_between(x, ceiling: int = 10):
+def _is_between(val, low: int = 1, high: int = 10):
     """
     Helper function that verifies if the input is below ceiling value
     """
-    return x in range(1, ceiling)
+    return low <= val < high
 
 
-def _add_random_noise(x, rng):
-    """
-    Helper function that adds random noise to entry according a Uniform distribution
-    """
-    return x + rng.uniform(0, 1)
+def _seed_to_rng(seed: Optional[Union[int, Generator]] = None) -> Generator:
+    if seed is None:
+        return np.random.default_rng()
+    if isinstance(seed, int):
+        return np.random.default_rng(seed)
```

Comment:
This can return None. I think you want to add `return seed` as the default?

Also, see comment above about `Generator`

Reply:
I think I understand why you would want to return `seed` in the case that we input a `Generator`, but not sure why we would want `None`